### PR TITLE
[Notifier] Documentation for Microsoft Teams Options

### PR DESCRIPTION
**Pull request by KamilKubicki**
Docs for symfony/symfony#40738

Replaces https://github.com/symfony/symfony-docs/pull/15232

### CODE REVIEW

**Comment by KamilKubicki** [comment]
Please review as complementary commit to:

[Notifier] Add docs for Microsoft Teams Options #15232

File: notifier/chatters.rst [hunk 1]
```diff
@@ -218,6 +218,92 @@ to add `message options`_::
...
+    $chatMessage = (new ChatMessage('Contribute To Symfony'))->transport('microsoftteams');
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/microsoft_teams/message.png
+   :align: center
+
+Adding Interactions to a Microsoft Teams Message
+-----------------------------------------
```
Comment (Nyholm) [inline notifier/chatters.rst]: There should be the same length. 

@OskarStark, shouldn't DOCtor find this?

File: notifier/chatters.rst [hunk 2]
```diff
@@ -218,6 +218,92 @@ to add `message options`_::
...
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\HttpPostAction;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\Input\DateInput;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\Input\TextInput;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsOptions;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsTransport;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Field\Fact;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Section;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    $chatMessage = new ChatMessage('');
```
Comment (Nyholm) [inline notifier/chatters.rst:254]: Should the first argument really be empty?

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. 

Could we add more "real" values in the examples?

File: notifier/chatters.rst (hunk 2 above)
Comment (KamilKubicki) [reply notifier/chatters.rst:254]: `new ChatMessage('') ` requires a subject string argument, if this argument is given, then `->text('Text')`  should be skipped while setting `new MicrosoftTeamsOptions()`. In otherw words `new ChatMessage('') ` and `->text('Text')` are equivalents. In my example I used second one to set the text.

File: notifier/chatters.rst (hunk 1 above)
Comment (KamilKubicki) [reply notifier/chatters.rst]: Fixed

File: notifier/chatters.rst [hunk 3]
```diff
@@ -256,31 +256,38 @@ to add `MessageCard options`_::
     // Action elements
     $input = new TextInput();
     $input->id('input_title');
-    $input->isMultiline(true)->maxLength(5)->title('Input Title');
+    $input->isMultiline(true)->maxLength(5)->title('In few words, why would you like to participate?');
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
    $input->isMultiline(true)->maxLength(5)->title('In a few words, why would you like to participate?');
```

File: notifier/chatters.rst [hunk 4]
```diff
@@ -256,31 +256,38 @@ to add `MessageCard options`_::
...
-        ->title('Title')
+        ->title('Symfony Online Meeting')
+        ->text('Symfony Online Meeting are the events where the best developers meet to share experiences...')
         ->summary('Summary')
         ->themeColor('#F4D35E')
-        ->section(
-            (new Section())
-            ->title('Section - name')
+        ->section((new Section())
+            ->title('Talk about Symfony 5.x - would you like to join? Please give a shout!')
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
            ->title('Talk about Symfony 5.3 - would you like to join? Please give a shout!')
```

File: notifier/chatters.rst [hunk 5]
```diff
@@ -256,31 +256,38 @@ to add `MessageCard options`_::
...
+            ->fact((new Fact())
+                ->name('Presenter')
+                ->value('John Doe')
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
                ->value('Fabien Potencier')
```

File: notifier/chatters.rst (hunk 1 above)
Comment (OskarStark) [reply notifier/chatters.rst]: > @OskarStark, shouldn't DOCtor find this?

AFAIK there is no such rule, because Sphinx allows this 🤔

**Comment by KamilKubicki** [comment]
Hi @Nyholm, related issue from @OskarStark was already merged - [Notifier] Microsoft Teams: JSON structure error fix for simple message #40834. Should you require any action from my side for this documentation or we can consider that it is ready for merge?

**Comment by OskarStark** [comment]
It's done but we can merge this for 5.4 when the feature got merged

**Comment by javiereguiluz** [comment]
Thanks Kamil for this nice contribution 🎉

**Pull request merged**